src/shared/components/plots/PlotsTab.tsx

```diff
@@ -220,6 +220,7 @@ export enum MutationCountBy {
     MutationType = 'MutationType',
     MutatedVsWildType = 'MutatedVsWildType',
     DriverVsVUS = 'DriverVsVUS',
+    VAF = 'VAF',
```

Comment:
Should we just spell it out for clarity?

```suggestion
    VariantAlleleFrequency = 'VariantAlleleFrequency',
```